build/patches/Bromite-AdBlockUpdaterService.patch

```diff
@@ -983,12 +1531,56 @@ new file mode 100644
 +  tasks_.erase(task);
 +}
 +
++std::string AdBlockUpdaterService::GetAdBlockFiltersURL() {
++  return pref_service_->GetString(kAdBlockFiltersURL);
++}
++
++void AdBlockUpdaterService::SetAdBlockFiltersURL(const std::string url) {
++  pref_service_->SetString(kAdBlockFiltersURL, url);
++  pref_service_->SetTime(kAdBlockLastCheckTime, base::Time());
++}
++
++std::string AdBlockUpdaterService::GetMostRecentIndexedVersion() {
++  auto version = ruleset_service_->GetMostRecentlyIndexedVersion();
++  return version.content_version;
++}
++
++long AdBlockUpdaterService::GetLastUpdate() {
++  return pref_service_->GetTime(kAdBlockLastCheckTime).ToJavaTime();
++}
++
++long AdBlockUpdaterService::GetLastOkUpdate() {
++  base::Time lastOk = pref_service_->GetTime(kAdBlockLastCheckTimeOk);
++  return lastOk.ToJavaTime();
++}
++
++int AdBlockUpdaterService::GetAdBlockUpdateTimeFrequency() {
++  return pref_service_->GetInteger(kAdBlockFiltersCheckFrequency);
++}
++
++void AdBlockUpdaterService::SetAdBlockUpdateTimeFrequency(int days) {
++  pref_service_->SetInteger(kAdBlockFiltersCheckFrequency, days);
```

Comment:
When selecting the `Never` option it looks like the update will still be started.

Reply:
it's correct, update does not start and the message is

chromium: [INFO:adblock_updater_service.cc(122)] AdBlockUpdaterService: user disabled.